src/MicrosoftTeams.schema.json

```diff
@@ -43,79 +43,123 @@
                 "termsOfUseUrl"
             ]
         },
-        "tabs": {
+        "name": {
+            "type": "string",
+            "description": "The display name of the app.",
+            "maxLength": 16
+        },
+        "description": {
+            "type": "object",
+            "properties": {
+                "short": {
+                    "type": "string",
+                    "description": "A short description of the app used when space is limited. Maximum length is 80 characters.",
+                    "maxLength": 80
+                },
+                "full": {
+                    "type": "string",
+                    "description": "The full description of the app. Maximum length is 256 characters.",
+                    "maxLength": 256
+                }
+            },
+            "required": [
+                "short",
+                "full"
+            ]
+        },
+        "icons": {
+            "type": "object",
+            "description": "Each icon image file must be a transparent PNG, with a white or light-colored background.",
+            "properties": {
+                "44": {
+                    "type": "string",
+                    "description": "An icon for the app sized to 44x44."
+                },
+                "88": {
+                    "type": "string",
+                    "description": "An icon for the app sized to 88x88."
+                }
+            },
+            "required": [
+                "44",
+                "88"
+            ]
+        },
+        "accentColor": {
+            "type": "string",
+            "description": "A color to use in conjunction with the app's icons. The value must be a valid HTML color code starting with '#', for example `#4464ee`."
+        },
+        "galleryTabs": {
             "type": "array",
             "items": {
                 "type": "object",
                 "properties": {
-                    "id": {
+                    "configUrl": {
```

Comment:
I suggest we call this configurationUrl since I'm not a fan of using abbreviations in public-facing schemas/APIs. What do you think?